tools/wake-format/emitter.cpp

```diff
@@ -28,7 +28,7 @@
 wcl::rope Emitter::newline(ctx_t ctx) {
   wcl::rope_builder builder;
 
-  builder.append("\n");
+  builder.append("⏎\n");
```

Comment:
What's up with this? Is this a carriage return or something? Is just only for debug?

Reply:
just debugging, I'll put it behind the debug flag. Can I do that in the next PR though?